fasta/faBin_test.go

```diff
@@ -0,0 +1,96 @@
+package fasta
+
+import (
+	"github.com/vertgenlab/gonomics/dna"
+	"log"
+	"testing"
+)
+
+var (
+	genome1     = "testdata/testBreakBinning.fa"
+	expBases1   = dna.StringToBases("ACGTA")
+	expBases2   = dna.StringToBases("CGT")
+	expBases3   = dna.StringToBases("AC")
```

Comment:
Can you help me understand why the third bin would have 'AC' as the bases and not ''ACGTA'?
Actually, I expected different functionality for all the bins. Maybe we can talk more in code review about this as well.